src/App.js

```diff
@@ -0,0 +1,20 @@
+import './App.css';
+import { Route, Routes } from 'react-router-dom';
+import Navbar from './components/Navbar';
+import BookList from './components/BookList';
+
+function App() {
+  return (
+    <div className="App">
+      <div className="book-sheet">
+        <Navbar />
+        <Routes>
+          <Route path="/" element={<BookList />} />
+          <Route path="/categories" />
```

Comment:
- kindly, Create a Category component and navigate this URL to the category component to full fill the page👍
`<Route path="/categories" element={<Categories />} />`